core/ui/compose/designsystem/README.md

```diff
@@ -36,3 +36,4 @@ This can save time and improve the overall quality, as well as facilitate collab
 
 - [Atomic Design Methodology | Atomic Design by Brad Frost](https://atomicdesign.bradfrost.com/chapter-2/)
 - [Atomic Design: Getting Started | Blog | We Are Mobile First](https://www.wearemobilefirst.com/blog/atomic-design)
+
```

Comment:
Is this just adding a newline at EOF, or a full empty line? The latter seems a bit weird to me, the former I support. I guess a flexmark rule?

Reply:
It's a flexmark rule to have a newline after the last list item + newline on end of file, not ideal but way faster than prettier before.